src/test/groovy/graphql/introspection/IntrospectionTest.groovy

```diff
@@ -353,4 +360,79 @@ class IntrospectionTest extends Specification {
         then:
         queryTypeFields == [[name: "inA"], [name: "inB"], [name: "inC"]]
     }
+
```

Comment:
Copied from "Issue 296" test file. Giving it a better home here in this IntrospectionTest file